staging/operator-lifecycle-manager/pkg/package-server/server/server.go

```diff
@@ -16,14 +16,29 @@ import (
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/util/workqueue"
 
+	operatorsv1 "github.com/operator-framework/api/pkg/operators/v1"
 	"github.com/operator-framework/operator-lifecycle-manager/pkg/api/client"
+	"github.com/operator-framework/operator-lifecycle-manager/pkg/api/client/clientset/versioned"
+	"github.com/operator-framework/operator-lifecycle-manager/pkg/api/client/informers/externalversions"
```

Comment:
nit: it's nice to alias this to `olminformers` or something similar, `externalversions` is a nondescript name

Reply:
I believe I took this from elsewhere, but I'm ok with aliasing.